core/facades/auth_facade.go

```diff
@@ -3,15 +3,13 @@ package facades
 import (
 	"context"
 
-	"github.com/PTSS-Support/identity-service/api/dtos/requests"
-	"github.com/PTSS-Support/identity-service/api/dtos/responses"
+	requests "github.com/PTSS-Support/identity-service/api/dtos/requests/auth"
+	responses "github.com/PTSS-Support/identity-service/api/dtos/responses/auth"
```

Comment:
again, why have requests and responses at the beginning here?

Reply:
Cause else no worky :neckbeard: